tests/fpga/fpga_stencil.py

```diff
@@ -3,7 +3,7 @@
 import dace
 
 
-def make_sdfg(name="fpga_stencil_test", dtype=dace.float32, veclen=8):
+def make_sdfg(name="fpga_stcl_test", dtype=dace.float32, veclen=8):
```

Comment:
What happened here? :tizianoeat:

Reply:
That would generate a long name, longer than the fantastic kernel name limit for Intel

Reply:
Hahaha I forgot about that, okay then